<?xml version="1.0"?>
<launch>
  <node pkg="flying_gazebo_stereo_cam" type="robot_interface" name="robot_interface" clear_params="true" output="screen">
    
    <param name="model_name" value="flying_stereo_cam" />
    <param name="camera_frame_name" value="cam_pos" /> <!--cam_pos-->
    <param name="world_frame_name" value="world" />
    <param name="sensor_in_topic" value="/camera/points2" /> <!--/camera/points2-->
    <param name="sensor_out_name" value="/world/pcl_input" />


    <param name="yb_model_name" value="youbot" />
    <param name="yb_eef_frame_name" value="arm_link_5" />
    <param name="yb_world_frame_name" value="base_footprint" />
    
  </node>
   
  <!--node pkg="rviz" type="rviz" name="rviz" clear_params="true" output="screen" args="-d $(find flying_gazebo_stereo_cam)/config/bunny.rviz"/-->
</launch>
